src/JPEGView/PNGWrapper.cpp

```diff
@@ -107,9 +107,12 @@ void BlendOver(unsigned char** rows_dst, unsigned char** rows_src, unsigned int
 }
 #endif
 
-void* PngReader::ReadNextFrame()
+void* PngReader::ReadNextFrame(void** exif_chunk, png_uint_32* exif_size)
```

Comment:
does this mean there can be different EXIF per frame of Png?!

Reply:
No, the `info_ptr` keeps track of the exif data until it gets destroyed, and this way we can get the Exif data on every frame, not just the first one. I could also just copy it to the cache like I did for JXL.